src/sqlancer/sqlite3/SQLite3Provider.java

```diff
@@ -58,47 +58,50 @@ public SQLite3Provider() {
     }
 
     public enum Action implements AbstractAction<SQLite3GlobalState> {
-        PRAGMA(SQLite3PragmaGenerator::insertPragma), //
-        INDEX(SQLite3IndexGenerator::insertIndex), //
-        INSERT(SQLite3InsertGenerator::insertRow), //
-        VACUUM(SQLite3VacuumGenerator::executeVacuum), //
-        REINDEX(SQLite3ReindexGenerator::executeReindex), //
-        ANALYZE(SQLite3AnalyzeGenerator::generateAnalyze), //
-        DELETE(SQLite3DeleteGenerator::deleteContent), //
+        PRAGMA(SQLite3PragmaGenerator::insertPragma), // 0
```

Comment:
Not sure why we are numbering the actions here?

Reply:
I number these actions for debugging. The enumeration variable only includes index information in execution, and these numbers help me to quickly locate which action is selected and executed.